Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -401,11 +432,11 @@ MediaPlayer::SupportsType MediaPlayerPrivateGStreamerMSE::supportsType(const Med
     if (!ok)
         height = 0;
 
-    if (width > MEDIA_MAX_WIDTH || height > MEDIA_MAX_HEIGHT)
+    if (width > mediaMaxWidth || height > mediaMaxHeight)
```

Comment:
If mediaMaxWidth is zero it means that there's no width limit. Same for height. The if should remain something like this:
```
if ((mediaMaxWidth && width > mediaMaxWidth) || (mediaMaxHeight && height > mediaMaxHeight))
```